### PR TITLE
Add schema content

### PR DESCRIPTION
**Pull request by nahteb**
We've created a [schema](https://github.com/alphagov/data-standards-authority/blob/main/adr/0001-standards-catalogue-data-structure.md) as part of our plans to introduce schema.org metata for the standards section, and the content in each of the standards should now match it.

### CODE REVIEW

**Review by jamietanna — CHANGES_REQUESTED** [review]
Can we please get the schema Rake task added to CI? Should now be safe to enforce it :+1:

File: standards-catalogue/source/layouts/standards_table.erb [hunk 1]
```diff
@@ -1,36 +1,52 @@
...
+          <strong><%= current_page.data.contraction %></strong>
+        </td>
+      </tr>
+
+      <tr>
+        <td>Topics</td>
+        <td>
+          <ul>
+            <% current_page.data.tags.each do |tag| %>
+              <li><strong><%= tag %></strong></li>
```
Comment (jamietanna) [inline standards-catalogue/source/layouts/standards_table.erb:25]: :stop_sign: this isn't showing the pretty `name` for the Tag

File: standards-catalogue/source/layouts/standards_table.erb [hunk 2]
```diff
@@ -1,36 +1,52 @@
...
+
+      <tr>
+        <td>Also Known As</td>
+        <td>
+          <strong><%= current_page.data.contraction %></strong>
+        </td>
+      </tr>
+
+      <tr>
+        <td>Topics</td>
```
Comment (jamietanna) [inline standards-catalogue/source/layouts/standards_table.erb:21]: Maybe?

```suggestion
        <td>Tags</td>
```

File: standards-catalogue/source/layouts/standards_table.erb [hunk 3]
```diff
@@ -1,36 +1,52 @@
...
+        <td>
+          <ul>
```
Comment (jamietanna) [inline standards-catalogue/source/layouts/standards_table.erb:23]: Are we happy with the indentation being off compared to the other entries in the table?
2021-11-15-153034_702x307_scrot

File: standards-catalogue/source/layouts/standards_table.erb [hunk 4]
```diff
@@ -1,36 +1,52 @@
...
-            </td>
-          <% else %>
-            <td nowrap><strong><%= value %></strong> </td>
-          <% end %>
-        </tr>
-      <% end %>
+      <tr>
+        <td>Name</td>
+        <td>
+          <strong><%=link_to current_page.data.name, current_page.data.specification_url %></strong>
```
Comment (jamietanna) [inline standards-catalogue/source/layouts/standards_table.erb:9]: Is the difference between `name` and `title` important?

File: standards-catalogue/source/layouts/standards_table.erb [hunk 5]
```diff
@@ -1,36 +1,52 @@
...
+            <% current_page.data.tags.each do |tag| %>
+              <li><strong><%= tag %></strong></li>
+            <% end %>
+          </ul>
+        </td>
+      </tr>
+
+      <tr>
+        <td>Maintained By</td>
+        <td><strong><%= display_organisation(current_page.data.maintainer_id) %></strong> </td>
```
Comment (jamietanna) [inline standards-catalogue/source/layouts/standards_table.erb:33]:
```suggestion
        <td><strong><%= display_organisation(current_page.data.maintainer_id) %></strong></td>
```

**Review by jamietanna — COMMENTED** [review]
Awesome, almost there :raised_hands:

File: standards-catalogue/source/layouts/standards_table.erb (hunk 4 above)
Comment (nahteb) [reply standards-catalogue/source/layouts/standards_table.erb]: no that was a mistake, title shouldn't be in there any more - have changed it to name

File: standards-catalogue/source/layouts/standards_table.erb (hunk 3 above)
Comment (nahteb) [reply standards-catalogue/source/layouts/standards_table.erb]: should now look like:

Screenshot 2021-11-15 at 16 02 58

File: standards-catalogue/source/layouts/standards_table.erb (hunk 1 above)
Comment (nahteb) [reply standards-catalogue/source/layouts/standards_table.erb]: amended in https://github.com/alphagov/data-standards-authority/pull/128/commits/982e605bf05a2c5e1874bbc9c7dff89a6654da98

File: content/standards/UKGemini/index.html.md [hunk 6]
```diff
@@ -1,20 +1,15 @@
...
-validFrom: Jun-18
-status: endorsed
-startDate: 2020-12-01
-dateAdded: 2020-12-16
-dateUpdated: 2020-12-16
+maintainer_id: geospatial-commission 
 licence_id: CC-BY-4.0
+endorsement_status_events:
+- status: endorsed
+  date: '2020-12-01'
```
Comment (jamietanna) [inline content/standards/UKGemini/index.html.md:12]: Can we use double quotes here? There was a comment from Arnau before about making it consistent, and I think double quotes was preferred

**Review by jamietanna — COMMENTED** [review]
Looks good :+1: Not sure if we want Charles/Arnau to have a look, but I'm happy with it aside from the quotes

**Review by jamietanna — APPROVED** [review]

File: content/standards/360giving/index.html.md [hunk 7]
```diff
@@ -1,14 +1,15 @@
...
-organisation: three-sixty-giving
-name: 360Giving Schema
-status: review
-dateAdded: 2021-02-02
-dateUpdated: 2021-02-02
+maintainer_id: three-sixty-giving
 licence_id: CC-BY-4.0
+endorsement_status_events:
+- status: review
+  date: "2021-02-02"
```
Comment (arnau) [inline content/standards/360giving/index.html.md:12]: I would suggest we use unquoted dates as defined by the [YAML spec](https://yaml.org/spec/1.2.2/#example-timestamps). That said I haven't checked whether the yaml parser we use has any issues with it.

Comment (arnau) [reply content/standards/360giving/index.html.md:12]: Perhaps something to consider for the future is having an agreed YAML linting process to converge in style?

**Review by arnau — APPROVED** [review]
Comment on dates (and Jamies' on quotes) it looks good to me 👍

File: content/standards/360giving/index.html.md (hunk 7 above)
Comment (jamietanna) [reply content/standards/360giving/index.html.md:12]: I believe this came out of issues where not using quotes caused problems, but I can't remember :sweat_smile:

And agreed, that would probably be ideal - I'll raise an issue

**Pull request merged**